test/systemtests/network_test.go

```diff
@@ -338,6 +338,7 @@ func (s *systemtestSuite) testNetworkAddDeleteTenant(c *C, encap, fwdmode string
 			}
 		}
 
+		time.Sleep(3 * time.Second)
```

Comment:
Could we check if the networks have been created instead? I'm not saying I have anything against this change, quite the opposite. It's just that we're using sleep in a lot of places and I'm not sure it's always right.